### ISSUE TITLE
Provide an alternative to `alt`+`enter` to validate forms

### ISSUE THREAD

**Issue by b-per** [pomdtr/sunbeam#91]
For some reason `opt` + `enter` on Mac doesn't work in the Warp Terminal while it works in the VSCode terminal and in iTerm2.

In Warp, it adds the value `enter` to the field. I have the config set to consider *Left option key as Meta* but it still doesn't work.

I don't know if it can be fixed on the sunbeam side and/or Warp side but another option would be to allow custom key bindings to validate forms.

**Comment by pomdtr** [comment]
Yep there is something really strange with the handling of the alt+enter shortcut in warp. 

Custom keybinding (for default shortcut) is a feature I would like to support at some point.

**Comment by smetroid** [comment]
Not sure if it's relevant, but as someone who comes from the UNIX world, and use Vim and Vim shortcuts for everything I do, I found a tool that allows me to change the keybindings globally or per application.  The idea is to leave the application default keybindings and shortcuts as they are and use[ keymapper ](https://github.com/houmain/keymapper) to change the shortcuts for specific applications.  This will allow you to modify and update one config file to be able to manipulate any application defined in the config.

Here is a link to keymapper, which is probably one of the best tools I've found that has made my life a lot lot better, from an efficiency/productivity point of view.
https://github.com/houmain/keymapper

And John Lindquist (creator of script kit) has a really good video on why you may want to leave shortcuts for the apps as they are and use something like keymapper or similar to change the shortcuts per app.
https://egghead.io/talks/egghead-save-your-hands-and-save-your-time-rethinking-how-to-use-a-keyboard

**Comment by joshua-Evans-1** [comment]
Hey! Just came across this issue and wanted to suggest a potential solution. I noticed you're using Charm's Bubble Tea framework for your TUI and keybindings.

Bubble Tea(charm) has an excellent component library called "Bubbles" that includes a [key binding management system](https://github.com/charmbracelet/bubbles?tab=readme-ov-file#key). This would allow users to define custom keymaps directly in your configuration file. When the config is parsed, it could automatically update the keybindings throughout the application - including any help menus where shortcuts are displayed.

This approach would provide a native solution for key remapping rather than requiring users to rely on external tools like keymapper. It would also maintain consistency across different terminals since the remapping would happen at the application level rather than depending on terminal-specific behavior.
